android/build.gradle

```diff
@@ -1,16 +1,27 @@
+buildscript {
+    repositories {
```

Comment:
I made these changes to get the project building standalone (outside the host react native repo). I used `npx create-react-native-library` and copied the `build.gradle` that was generated